src/button/button-group.scss

```diff
@@ -10,16 +10,22 @@
   $childSelector: 'input, button'; // target nested inputs and buttons
 
   > * {
+    display: flex;
```

Comment:
why not to child (child selector) ?

Reply:
z-index is applied to "flex items" which means children of the element that has `display: flex`

Reply:
hopefully this will not break anything....
Did you figure why in css it was not reproduced?

Reply:
It will not break anything because this div only has one child, so `display: flex` does not hurt (it actually helps in most cases, like vertical alignment)

I could not find out exactly why, but I think it might have to do with Storybook's custom styling. If I popped the story out in new tab, it was working fine. It's possible that user's app could have similar styling which might break it, so it's better to fix it anyway

Reply:
Actually I thought about it some more and found three situations:
- split-menu/tooltips are put in the same div by default, so it doesn't always have "only one child". But those are using absolute/fixed positioning so it should not be affected
- input-container has input nested inside a div, so it's not a direct child but it's still a grid item, so z-index works.
- there is one situation it could "break", which is if the child is wrapped (e.g. a div around a disabled button). In that case, the user would need to manually add `display: flex` or `position: relative`